models/DataObject/Classificationstore/KeyConfig.php

```diff
@@ -61,7 +61,7 @@ final class KeyConfig extends Model\AbstractModel
 
     protected ?int $modificationDate = null;
 
-    protected string $definition;
+    protected string $definition = '[]';
 
     protected bool $enabled;
```

Comment:
A default value should also be specified here:
```suggestion
    protected bool $enabled = false;
```

Reply:
Agreed, though it wasn't causing issues yet for consistency it would be better to include it as well. Thanks!